bundle/run.sh

```diff
@@ -244,10 +255,48 @@ while true; do
 done
     
 # authorization
+## configuration
+### setup root-cas
+while read -r line; do
```

Comment:
please create a default yaml file for authorization service and store it to data folder, so it's easy to map a volume, modify it / review it / add your own one.

Reply:
not possible - we are configure via ENV. we can do it but after we migrate all cfgs.